doc/nep1.rst

```diff
@@ -301,3 +301,13 @@ Miscellaneous
     let a = """foo
     bar
     """
+
+- A getter API for a private field `foo` should preferably be named `foo`, not `getFoo`.
+  A getter-like API should preferably be named `getFoo`, not `foo` if:
+    * the API has side effects
+    * or the cost is not `O(1)`
+  For in between cases, there is no clear guideline.
```

Comment:
Should be better to think on a clear guideline, because this is the official documentation... :thinking:

Reply:
@juancarlospaco you can't decide for all cases, Nim is not the "one way do to it" language

Reply:
I feel the `O(1)` may be too specific, maybe wording it like `if it is computationally costly name it getFoo` or similar.

Reply:
That's the whole point, O(1) vs not O(1) though. Builtin field accessors are O(1), and so should procs mascarading as field accessors.